### DISCUSSION TITLE
V2 preview can't open

### DISCUSSION THREAD

**Discussion by wilding96** [vbenjs/vue-vben-admin#4396] [Q&A]
url:
https://v2.vben.pro/#/login?redirect=/dashboard/analysis
Q：
Request failed with status code 404
eq.
![image](https://github.com/user-attachments/assets/2cfa9a5f-2015-48eb-9e0e-120453685104)

**Comment by vince292007** [comment]
<img width="1456" alt="image" src="https://github.com/user-attachments/assets/10729acd-1186-4203-9328-6eb6333ce614">

**Reply by wilding96** [reply]
after reset my chrome setting，it's worked! thank you!
